downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.5.1 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201-5-1/
+active: 2201-5-1
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-5-1
+    - /downloads/swan-lake-release-notes/2201.5.1/
+    - /downloads/swan-lake-release-notes/2201-5-1-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-5-1-swan-lake
+---
+
+## Overview of Ballerina Swan Lake 2201.5.1
+
+<em>Swan Lake 2201.5.1 is the first patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the language, tooling, and standard library.</em>
```

Comment:
```suggestion
<em>Swan Lake 2201.5.1 is the first patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the language and standard library.</em>
```
Check if we have tooling updates going in this release and remove.

Reply:
will remove later if needed.